### DISCUSSION TITLE
Q: 请问如何禁用Waline Example页面的展示？ || Q: How to disable the display of the Waline Example page?

### DISCUSSION THREAD

**Discussion by LxnChan** [walinejs/waline#1976] [Q&A]
### Describe the bug | 问题描述

想通过nginx的规则屏蔽掉打开waline域名时的Waline Example页面，请问如何操作？

### Website URL | 问题网站

N/A

### Where your waline deploy? | 服务部署在哪里？

Docker

### Where your comment data store? | 数据存储在哪里？

MySQL

---
### Describe the bug | Problem description

I want to block the Waline Example page when opening the waline domain name through nginx rules, how to do it?

### Website URL | Problem Website

N/A

### Where your waline deploy? | Where is the service deployed?

Docker

### Where your comment data store? | Where is your comment data store?

MySQL

**Comment by lizheming** [comment]
我认为你已经有思路了，nginx 的具体配置规则可以咨询下 chatgpt ？ 

---
I think you already have an idea. For the specific configuration rules of nginx, you can consult chatgpt?
